instrumentation/netty/netty-4.1/library/src/main/java/io/opentelemetry/instrumentation/netty/v4_1/NettyServerTelemetryBuilder.java

```diff
@@ -46,13 +49,32 @@ public NettyServerTelemetryBuilder setCapturedResponseHeaders(
     return this;
   }
 
+  /**
+   * Configures the instrumentation to recognize an alternative set of HTTP request methods.
+   *
+   * <p>By default, this instrumentation defines "known" methods as the ones listed in <a
+   * href="https://www.rfc-editor.org/rfc/rfc9110.html#name-methods">RFC9110</a> and the PATCH
+   * method defined in <a href="https://www.rfc-editor.org/rfc/rfc5789.html">RFC5789</a>.
+   *
+   * <p>Note: calling this method <b>overrides</b> the default known method sets completely; it does
+   * not supplement it.
+   *
+   * @param knownMethods A set of recognized HTTP request methods.
+   */
+  @CanIgnoreReturnValue
+  public NettyServerTelemetryBuilder setKnownMethods(Set<String> knownMethods) {
+    this.knownMethods = knownMethods;
+    return this;
+  }
```

Comment:
does it make any sense to have `HttpClientTelemetryBuilder` interface to help enforce that these implementations are consistent (and maybe we can inherit javadoc)?

Reply:
> (and maybe we can inherit javadoc)?

Hmm, we can put a `@see` tag I think. I don't think we can actually enforce that though.